docs/docsite/rst/network/user_guide/network_working_with_command_output.rst

```diff
@@ -63,3 +66,34 @@ The ``wait_for`` argument must always start with result and then the
 command index in ``[]``, where ``0`` is the first command in the commands list,
 ``1`` is the second command, ``2`` is the third and so on.
 
+
+Handling prompts in network modules
+===================================
+
+Network devices may require that you answer a prompt before performing a change on the device. Individual network modules such as :ref:`ios_command <ios_command_module>` and :ref:`nxos_command <nxos_command_module>` can handle this with a ``prompt`` parameter.
+
+You can also use the :ref:`cli_command <cli_command_module>` to handle multiple prompts.
+
+.. code-block:: yaml
+
+  ---
+   - name: reboot ios device
+     cli_command:
+       command: reload
+       prompt:
```

Comment:
`prompt` is python regex, based on the past issues people often add special characters like `?` in the prompt value which results in prompt not matching and timeout. To avoid this we can highlight in bold that the prompt value should be a python regex that matched the actual prompt. The special characters should be handled correctly in the prompt regex.

Reply:
This is, I think, the most important thing to clarify wrt the `prompt` argument.

Reply:
added a note to cover this...